scaffolder-templates/add-tenant-to-alerts/template.yaml

```diff
@@ -67,6 +67,26 @@ spec:
       action: debug:log
       input:
         message: 'alertable_tenants.json content: ${{ steps["parse-alerts"].output.content }}'
+    - id: add-customer-tag
```

Comment:
We should probably rename this scaffolder template to something that indicates its finalizing a customer.

Reply:
Good point, updating the title and description.